infra/aws/terraform/registry-sandbox-k8s-io-image-layers/versions.tf

```
@@ -0,0 +1,24 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+This file defines:
+- Required Terraform version
+*/
+
+terraform {
+  required_version = "~> 1.1.0"
```

```suggestion
  required_version = "~> 1.1.2"
```
Early patch releases in this series have a bug - details in https://github.com/hashicorp/terraform/releases/tag/v1.1.2
I would pin to a version that has that bug fixed.

---

I use `tfswitch` locally. it will detect the version and automatically pick the latest version. I'm not sure we want to pick a specific version.

---

I do recommend this pin. We can't be sure that another contributor is also using the same `tfswitch` setup.

---

Fixed